core/src/test/scala-3/cats/derived/OrderSuite.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2015 Miles Sabin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.derived
+
+import cats.Order
+import cats.kernel.laws.discipline.{OrderTests, SerializableTests}
+import org.scalacheck.Arbitrary
+
+class OrderSuite extends KittensSuite {
+  import OrderSuite._
+  import TestDefns._
+
+  def testOrder(context: String)(using
```

Comment:
Same here - we can simplify by making this `inline` like with your `Apply` PR

Reply:
Here we don't need any Isomorphism and inline trick works great 👍